packages/dates/src/config.ts

```diff
@@ -60,15 +64,17 @@ export const seasons: SeasonConfig[] = [
     starterNftAddress: '0x0000000000000000000000000000000000000000' as Address,
     standardNftAddress: '0x0000000000000000000000000000000000000000' as Address,
     weeksPerSeason: 1,
-    draft: true
+    draft: true,
+    allocatedTokens: 0
   },
   // Season 1
   {
     start: '2025-W18', // April 28th 2025
     title: 'Season 1',
-    starterNftAddress: '0x77ef845f8b2b7b40b68af10d1031313983ccf5a2' as Address,
-    standardNftAddress: '0x1aa94658c5586284bb7815e590a3456f76901500' as Address,
-    weeksPerSeason: 13
+    starterNftAddress: '0x9b11a12f267b21580ef911e404e96659d27eef84' as Address,
+    standardNftAddress: '0xa32f8737513454d6a938359614fdf47838a2b6d7' as Address,
+    weeksPerSeason: 13,
+    allocatedTokens: 2500
```

Comment:
shouldn't this be 	1,200,000? https://scoutgame.xyz/info/dev-tokens

Reply:
all of these needs (the contracts and allocated tokens) to be changed. I used 2500 for testing purposes. I'm gonna have a different config for onchain